backend/db/utils.py

```diff
@@ -157,16 +157,22 @@ def is_table_up_to_date(table_name: str, skip_if_updated_within_hours: int = Non
     last_updated_key = f'last_updated_{table_name}'
     return check_if_updated(last_updated_key, skip_if_updated_within_hours)
 
-
 # todo: Can update update_db_status_var() so that it can accept optional param 'con' to improve performance.
 def update_db_status_var(key: str, val: str, local=False):
     """Update the `manage` table with information for a given variable, e.g. when a table was last updated
     todo: change to a 1-liner UPDATE statement"""
     with get_db_connection(schema='', local=local) as con:
         run_sql(con, f"DELETE FROM public.manage WHERE key = '{key}';")
-        sql_str = f"INSERT INTO public.manage (key, value) VALUES (:key, :val);"
```

Comment:
@hugwuoke Uhoh. I think when you were pair programming with Siggie you guys might have seen that I had a todo here "change to a 1-liner UPDATE statement". It looks like you guys were trying to do that, and ended up deleting the line where it inserts a new value for the db var, but only leaving the 1 line where it deletes the variable. When that change was made, I think it was meant to be changed from a `DELETE` to an `UPDATE`.

It probably doesn't take very long to change it fo an `UPDATE`, but I'm a little strapped for time here today as well, so I'm just going to revert it back to the way it was before: delete the old variable, insert the variable again w/ its new value.